app/economy/management/commands/get_prices.py

```diff
@@ -110,7 +110,7 @@ def polo():
     for b in Bounty.objects.all():
         print('refreshed {}'.format(b.pk))
         try:
-            b._val_usd_db = b.value_in_usdt
+            b._val_usd_db = b.value_in_usdt_now
```

Comment:
same comment here as the above one on `_val_usd_db`

Reply:
ok.

Reply:
done.